packages/firestore/test/lite/integration.test.ts

```diff
@@ -2402,3 +2404,311 @@ describe('Count quries', () => {
+        `abc/123/zzz/${collectionGroupId}`
+      ];
+      const batch = writeBatch(db);
+      for (const docPath of docPaths) {
+        batch.set(doc(db, docPath), { x: 1 });
+      }
+      await batch.commit();
+      const snapshot = await getAggregate(
+        collectionGroup(db, collectionGroupId),
+        { myCount: count() }
+      );
+      expect(snapshot.data().myCount).to.equal(2);
+    });
+  });
+
+  it('aggregateQuerySnapshotEqual on same queries be truthy', () => {
+    const testDocs = [
+      { author: 'authorA', title: 'titleA' },
+      { author: 'authorA', title: 'titleB' },
+      { author: 'authorB', title: 'titleC' }
+    ];
+    return withTestCollectionAndInitialData(testDocs, async coll => {
+      const query1 = query(coll, where('author', '==', 'authorA'));
+      const query2 = query(coll, where('author', '==', 'authorA'));
+      const snapshot1A = await getAggregate(query1, { myCount: count() });
+      const snapshot1B = await getAggregate(query1, { myCount: count() });
+      const snapshot2 = await getAggregate(query2, { myCount: count() });
+      expect(aggregateQuerySnapshotEqual(snapshot1A, snapshot1B)).to.be.true;
+      expect(aggregateQuerySnapshotEqual(snapshot1A, snapshot2)).to.be.true;
+    });
+  });
+
+  it('aggregateQuerySnapshotEqual on same queries with different documents size be falsy', () => {
+    const testDocs = [
+      { author: 'authorA', title: 'titleA' },
+      { author: 'authorA', title: 'titleB' },
+      { author: 'authorB', title: 'titleC' }
+    ];
+    return withTestCollectionAndInitialData(testDocs, async coll => {
+      const query1 = query(coll, where('author', '==', 'authorA'));
+      const snapshot1A = await getAggregate(query1, { myCount: count() });
+      await addDoc(coll, { author: 'authorA', title: 'titleD' });
+      const query2 = query(coll, where('author', '==', 'authorA'));
+      const snapshot1B = await getAggregate(query1, { myCount: count() });
+      const snapshot2 = await getAggregate(query2, { myCount: count() });
+      expect(aggregateQuerySnapshotEqual(snapshot1A, snapshot1B)).to.be.false;
+      expect(aggregateQuerySnapshotEqual(snapshot1A, snapshot2)).to.be.false;
+    });
+  });
+
+  it('aggregateQuerySnapshotEqual on different queries be falsy', () => {
+    const testDocs = [
+      { author: 'authorA', title: 'titleA' },
+      { author: 'authorA', title: 'titleB' },
+      { author: 'authorB', title: 'titleC' },
+      { author: 'authorB', title: 'titleD' }
+    ];
+    return withTestCollectionAndInitialData(testDocs, async coll => {
+      const query1 = query(coll, where('author', '==', 'authorA'));
+      const query2 = query(coll, where('author', '==', 'authorB'));
+      const snapshot1 = await getAggregate(query1, { myCount: count() });
+      const snapshot2 = await getAggregate(query2, { myCount: count() });
+      expect(aggregateQuerySnapshotEqual(snapshot1, snapshot2)).to.be.false;
+    });
+  });
```

Comment:
no action needed now, but just to keep in mind for next PR when emulator supports sum/avg:
TODO(sum/avg): we should add tests for `getAggregate with same aggregations be truthy` and `getAggregate with different aggregations be falsy`

Reply:
I think this is covered on line 2617, but please correct me if I'm wrong.

Reply:
yeah, I was thinking of different kinds of aggregations. e.g.

```ts
const snapshot1 = await getAggregate(query1, { myCount: count(), mySum: sum() });
const snapshot2 = await getAggregate(query1, { myCount: count(), mySum: sum() });
const snapshot3 = await getAggregate(query1, { mySum: sum(),     myAvg: avg() });

expect(aggregateQuerySnapshotEqual(snapshot1, snapshot2)).to.be.true;
expect(aggregateQuerySnapshotEqual(snapshot1, snapshot3)).to.be.false;
```
just remembered this doesn't need the emulator actually